tinytuya/wizard.py

```diff
@@ -138,6 +138,18 @@ def wizard(color=True, retries=None, forcescan=False, nocloud=False, quicklist=F
         with open(DEVICEFILE, "r") as infile:
             tuyadevices = json.load( infile )
     else:
+        if 'apiDeviceID' in config and config['apiDeviceID'] == 'scan':
```

Comment:
Should we make that `config['apiDeviceID'].lower() == 'scan':` to ignore user entered caps?

Reply:
Done!